static/app/components/avatar/avatarList.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import {Fragment, useEffect} from 'react';
+
+import AvatarList from 'sentry/components/avatar/avatarList';
+import Placeholder from 'sentry/components/placeholder';
+import JSXProperty from 'sentry/components/stories/jsxProperty';
+import Matrix from 'sentry/components/stories/matrix';
+import SizingWindow from 'sentry/components/stories/sizingWindow';
+import storyBook from 'sentry/stories/storyBook';
+import {useMembers} from 'sentry/utils/useMembers';
+import {useUserTeams} from 'sentry/utils/useUserTeams';
+
+function useLoadedMembers() {
+  const {members, loadMore, ...rest} = useMembers();
+
+  useEffect(() => {
+    // `loadMore` is not referentially stable, so we cannot include it in the dependencies array
+    loadMore();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  return {members, loadMore, ...rest};
+}
+
+export default storyBook(AvatarList, story => {
+  story('Default', () => {
+    const {teams, isLoading} = useUserTeams();
+    const {members, fetching} = useLoadedMembers();
+
+    return (
+      <Fragment>
+        <p>
+          Your Teams
+          <SizingWindow display="block" style={{width: '50%'}}>
+            {isLoading ? (
+              <Placeholder />
+            ) : (
+              <AvatarList teams={teams} typeAvatars="teams" />
```

Comment:
could make a note that `typeAvatars` prop is what's shown in the overflow tooltip